test/hs/Test/Ganeti/HTools/Node.hs

```diff
@@ -134,6 +134,7 @@ genEmptyOnlineNode =
                        , Node.rMemForth = 0
                        , Node.pRem = 0
                        , Node.pRemForth = 0
+                       , Node.xMem = 0
```

Comment:
Unfortunately we can't. I tried to, but then I realized, that hbal does a lot of placement simulations when finding optimal cluster allocation. During these steps the state of record is updated, but the state of world obviously isn't. At that point the function would give us different values. That's why we need to calculate the difference between state-of-world/state-of-record beforehand and use this value to predict what the state-of-world would be after those placements happen.